hamocc/mo_param1_bgc.F90

```diff
@@ -24,247 +24,364 @@ MODULE mo_param1_bgc
 !
 !  Patrick Wetzel    *MPI-Met, HH*    01.09.03
 !
-!  
+!
 !  Modified
 !  --------
 !  J.Schwinger,        *NORCE Climate, Bergen*    2020-05-26
 !
-!  - To facilitate easier use of 'only-lists' in use statements, make indices 
+!  - To facilitate easier use of 'only-lists' in use statements, make indices
 !    always defined also in case they are inside a #ifdef directive.
-!    
+!
 !
 !  Purpose
 !  -------
 !  - definition of indices in tracer arrays
 !
 !******************************************************************************
+     !use mo_control_bgc, only: use_cisonew
+      use mo_control_bgc
+
       implicit none
-      
-      INTEGER, PARAMETER :: ks=12,ksp=ks+1    ! ks: nb of sediment layers
+      public
 
+      INTEGER, PARAMETER :: ks=12,ksp=ks+1    ! ks: nb of sediment layers
       REAL,    PARAMETER :: safediv = 1.0e-25 ! added to the denominator of isotopic ratios (avoid div. by zero)
 
```

Comment:
Two things: I would suggest to give at least one comment per group (e.g.: `! Base tracer` etc. Then, is there any reason not to at least add a `protect` statement for the indices values?